packages/circuit-ui/components/Step/examples/YesOrNoSlider.tsx

```diff
@@ -113,7 +112,13 @@ export default function YesOrNoSlider({
               swipe={swipe}
             />
           </Swipeable>
-          <ButtonGroup align={'center'}>
+          <div
+            css={(theme) => css`
+              display: flex;
+              justify-content: center;
+              gap: ${theme.spacings.mega};
+            `}
+          >
```

Comment:
Note: this is an example of replacing the `ButtonGroup` by a custom buttons group when the new `ButtonGroup` styles (more opinionated than before) don't match intended designs